src/manager.js

```diff
@@ -270,3 +272,22 @@ Manager.prototype.processOnEnd = function (evt) {
     var index = self.nipples.indexOf(nipple);
     self.nipples.splice(index, 1);
 };
+
+// Remove destroyed nipple from the list
+Manager.prototype.ondestroyed = function(evt, nipple) {
+    nipple = this.nipples.get(nipple.identifier);
```

Comment:
This is very weird. 
Can you add a comment to explain it above the line `nipple = this.nipples.get[...]` to act as a reminder for me, please. 
Can you also log an issue about this with the info you've collected please?

I'll look at this later on. This is not a blocker for your PR, if your patch is working correctly.

After your comment, I'll be able to merge it. 
:+1: Good job and thank you very much for your help !